addons/calendar_sms/models/calendar.py

```diff
@@ -28,6 +29,8 @@ def _do_sms_reminder(self):
             )
 
     def action_send_sms(self):
+        if not self.partner_ids:
+            raise UserError(_("There are no attendees on these events"))
```

Comment:
Initial comment:
Why doesn't this one raise an error if not recipients ? Otherwise this one seems good as it targets partners.